src/components/Footer.jsx

```diff
@@ -1,20 +1,175 @@
 import React from 'react'
+import { Link } from 'gatsby'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
 import styled from 'styled-components'
-import logo from '../images/cfc_logo_short.svg'
+import longArrowAltRight from '../images/join-us/long-arrow-alt-right.svg'
 
 const StyledFooter = styled.footer`
-  width: 14rem;
+  width: 100%;
+  padding: 1rem 0;
+  display: block;
+  margin: 0 auto;
+
+  @media (min-width: 1200px) {
```

Comment:
This isn't something I like to do to keep things organized. I move all media queries to the bottom of the styled component. This makes them easier to find and also allows us al to be aware of how many and where all the breakpoints are. 

Let's gather them all and move them to the bottom (I also like to move them from largest to smallest breakpoint).